src/components/hashtag-list.tsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useLayoutEffect, useRef, useState } from 'react'
+
+import { Chip } from '@/components'
+import cn from '@/utils/cn'
+import { ClassName } from '@/models/interface'
+
+interface HashtagListProps extends ClassName {
+  placeId: string
+  hashtags: string[]
+}
+
+const HashtagList = ({ placeId, hashtags, className }: HashtagListProps) => {
+  const [visibleHashtags, setVisibleHashtags] = useState(hashtags)
+  const containerRef = useRef<HTMLDivElement>(null)
+
+  const useIsomorphicLayoutEffect =
+    typeof window !== 'undefined' ? useLayoutEffect : useEffect
+
+  const getFitContainerWidthHashtag = (
+    hashtags: string[],
```

Comment:
'hashtags' is already declared in the upper scope on line 12 column 33.eslint[@typescript-eslint/no-shadow](https://typescript-eslint.io/rules/no-shadow)